eng/Generation.psm1

```diff
@@ -0,0 +1,63 @@
+$repoRoot = Resolve-Path (Join-Path $PSScriptRoot '..')
+$autorestBinary = Join-Path $repoRoot 'node_modules' '.bin' 'autorest-beta'
+$AutorestPluginProject = Resolve-Path (Join-Path $repoRoot 'src' 'AutoRest.CSharp.V3')
+
+function Invoke($command)
+{
+    Write-Host "> $command"
+    pushd $repoRoot
+    if ($IsLinux)
```

Comment:
If we're adding Linux support, we have to make sure we use `Join-Path`/`Resolve-Path` in all places when accessing/writing files, as those cmdlets will use Environment.PathSeparator. Additionally, Linux file/folder names are case sensitive, so we have to make sure that we have those path elements cased exactly what they are on disk.

Reply:
It's not required, the pipeline runs just fine on linux and PS is normalizing the paths, I switched back to windows because of https://github.com/Azure/autorest/issues/3474